db/models/applications.py

```diff
@@ -0,0 +1,11 @@
+from db import db
+from db.models.account import Account
+from sqlalchemy import ForeignKey
+
+
+class AccountApplicationRelationship(db.Model):
```

Comment:
is this the best name - suppose we need to settle on a convention. maybe one for the DIP 🤓

Reply:
Agreed, shall I keep it as so for now? :)